examples/sealed-auctions/src/lib.rs

```diff
@@ -0,0 +1,382 @@
+#![feature(proc_macro_hygiene)]
+
+pub mod types;
+
+extern crate chrono;
+
+use chrono::Utc;
+use std::collections::{HashMap, HashSet}; // `use` statements can go anywhere
```

Comment:
```suggestion
use std::collections::{HashMap, HashSet};
```
This concept was covered in the previous tutorial.

Reply:
Done.